### PR TITLE
Update ci version

### PR DESCRIPTION
**Pull request by fengshunli**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -21,11 +21,12 @@ jobs:
...
-        uses: azure/setup-helm@v2.0
+        uses: azure/setup-helm@v3
         with:
-          version: v3.8.1
+          version: latest
+          token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.4.0
+        uses: helm/chart-releaser-action@v1
```
Comment (rambohe-ch) [inline .github/workflows/release.yaml:30]: v1 --> v1.5.0

File: .github/workflows/superlinter.yml [hunk 2]
```diff
@@ -16,9 +21,9 @@ jobs:
           fetch-depth: 0
 
       - name: Lint Code Base
-        uses: docker://github/super-linter:slim-v4
+        uses: github/super-linter@v4
         env:
-          DEFAULT_BRANCH: main
+          DEFAULT_BRANCH: master
```
Comment (rambohe-ch) [inline .github/workflows/superlinter.yml:21]: there is not master branch in repo.

File: .github/workflows/superlinter.yml [hunk 3]
```diff
@@ -3,7 +3,12 @@ name: Lint Code Base
 # Documentation:
 # https://help.github.com/en/articles/workflow-syntax-for-github-actions
 
-on: pull_request
+on:
+  push:
+    branches-ignore: [ master, main ]
+    # Remove the line above to run when pushing to master
+    pull_request:
+      branches: [ master, main ]
```
Comment (rambohe-ch) [inline .github/workflows/superlinter.yml]: remove `master`?

File: .github/workflows/superlinter.yml [hunk 4]
```diff
@@ -3,7 +3,12 @@ name: Lint Code Base
 # Documentation:
 # https://help.github.com/en/articles/workflow-syntax-for-github-actions
 
-on: pull_request
+on:
+  push:
+    branches-ignore: [ master, main ]
```
Comment (rambohe-ch) [inline .github/workflows/superlinter.yml]: remove this line?

File: .github/workflows/superlinter.yml (hunk 2 above)
Comment (fengshunli) [reply .github/workflows/superlinter.yml:21]: DEFAULT_BRANCH | master | The name of the repository default branch.
-- | -- | --

File: .github/workflows/superlinter.yml (hunk 4 above)
Comment (fengshunli) [reply .github/workflows/superlinter.yml]: done

**Pull request closed without merging**